powershell-executor/powershell-executor.Dockerfile

```diff
@@ -0,0 +1,21 @@
+ARG IMAGE_TAG=lts-7.2-ubuntu-22.04
+
+FROM mcr.microsoft.com/powershell:${IMAGE_TAG}
+
+# Use root user for installation
+USER root
+
+# Install Git
+RUN apt-get update && apt-get install -y git
+
+ARG DEBIAN_FRONTEND=noninteractive
+RUN apt install -y -f postgresql postgresql-contrib
+
+# Change default shell to powershell
+SHELL ["pwsh", "-Command"]
+
+# Install Az Module in Powershell
+RUN pwsh -Command Install-Module -Name Az -Repository PSGallery -Force -AllowClobber
```

Comment:
@ppotturi Since this is a wrapper docker image which is going to execute any ps script provided, it is better to put the common module imports in the docker file. else we will have to import it in all scripts where ever we use it.